python/mxnet/optimizer/optimizer.py

```diff
@@ -781,6 +784,240 @@ def update(self, index, weight, grad, state):
         ftml_update(weight, grad, prev_d, prev_v, prev_z, out=weight,
                     lr=lr, wd=wd, **kwargs)
 
+@register
+class SGDwFastLARS(Optimizer):
+    def __init__(self, momentum=0.0, lazy_update=True, lars_eta=0.001, lars_eps=0,
```

Comment:
could you add documentation like here - https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/optimizer/optimizer.py#L727

Reply:
Definitely

Reply:
Is this good enough ?